### PR TITLE
Updates the Readme

### PR DESCRIPTION
**Pull request by RealityOverseer**
This is SUPER basic and not at all intended to be the end all for the readme. I believe it would be better if we have at least something to help lead people to the community or make it known where our wiki and discord is.

Please tell me if you have any changes or if you are a maintainer feel free to make the changes yourself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-"Ne participes, nec mors sequetur"
+# Blackstone
+Blackstone is a fantasy roleplaying game based on the source code for Roguetown, and is inspired by Dungeons & Dragons. The goal of Blackstone is to provide a freeform fantasy roleplay environment with minimal admin intervention, where players are empowered to collectively decide what is and is not acceptable through in-game action.
```
Comment (Contrabang) [inline README.md:2]:
```suggestion
Blackstone is a fantasy roleplaying game based on the source code for Roguetown, and is inspired by Table Top Roleplaying Games. The goal of Blackstone is to provide a freeform fantasy roleplay environment with minimal admin intervention, where players are empowered to collectively decide what is and is not acceptable through in-game action.
```

Comment (RealityOverseer) [reply README.md:2]: Also make this fix on the wiki then, since this was directly copied and pasted from there

**Comment by JTGSZ** [comment]
leave the prev readmes one liner at the top

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-"Ne participes, nec mors sequetur"
+### "Ne participes, nec mors sequetur"
```
Comment (JTGSZ) [inline README.md:1]: don't make it big, jus leave it as "Ne participes, nec mors sequetur"

Comment (RealityOverseer) [reply README.md:1]: 💀

**Pull request merged**